tests/integration/test_openstack_builder.py

```diff
@@ -157,11 +182,34 @@ async def ssh_connection_fixture(
 
 @pytest.mark.amd64
 @pytest.mark.arm64
-@pytest.mark.usefixtures("cli_run")
+@pytest.mark.usefixtures("make_dangling_resources")
 async def test_run(ssh_connection: SSHConnection, dockerhub_mirror: str | None):
     """
     arrange: given openstack cloud instance.
     act: when run (build image) is called.
     assert: an image snapshot of working VM is created with the ability to run expected commands.
     """
     helpers.run_openstack_tests(dockerhub_mirror=dockerhub_mirror, ssh_connection=ssh_connection)
+
+
+@pytest.mark.amd64
+@pytest.mark.arm64
+async def test_openstack_state(
+    openstack_metadata: types.OpenstackMeta, test_id: str, image_config: types.ImageConfig
+):
+    """
+    arrange: given CLI run after dangling OpenStack resources creation.
```

Comment:
`make_dangling_resources` is not requested here. It has module scope, so I assume it is triggered by `test_run' requesting it. Is this correct? This would make this test dependent on `test_run` to be executed. Can we comment this more explicit here?

Reply:
Done, thank you for the reviow